### PR TITLE
* #272: OpenInfoMan - add compose for starting the service

### PR DESCRIPTION
**Pull request by areklalo**

### CODE REVIEW

File: script.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```
Comment (pgesek) [inline script.sh]: Rename to cmd.sh

File: compose/.env [hunk 2]
```diff
@@ -0,0 +1 @@
+OPENINFOMAN_URL=http://localhost:8984
```
Comment (pgesek) [inline compose/.env]: We can probably drop the .env for the local deployment, since we have a default.

**Review by pgesek — CHANGES_REQUESTED** [review]

File: compose/.env (hunk 2 above)
Comment (areklalo) [reply compose/.env]: Ok, I deleted this .env

File: script.sh (hunk 1 above)
Comment (areklalo) [reply script.sh]: Corrected.

**Review by pgesek — APPROVED** [review]

**Pull request merged**